src/main/java/ch/powerunit/extensions/matchers/provideprocessor/fields/OptionalFieldDescription.java

```diff
@@ -19,31 +19,91 @@
  */
 package ch.powerunit.extensions.matchers.provideprocessor.fields;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
+import java.util.Optional;
 
 import ch.powerunit.extensions.matchers.provideprocessor.ProvidesMatchersAnnotatedElementData;
+import ch.powerunit.extensions.matchers.provideprocessor.ProvidesMatchersAnnotatedElementMirror;
+import ch.powerunit.extensions.matchers.provideprocessor.RoundMirror;
+import ch.powerunit.extensions.matchers.provideprocessor.RoundMirrorSupport;
 
 public class OptionalFieldDescription extends DefaultFieldDescription {
 
+	private final RoundMirror roundMirror;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'roundMirror' could be replaced by a local variable.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5550779)